webrob/templates/flask_user/user_data.html

```diff
@@ -2,38 +2,35 @@
   
 {% block head %}
 <style>
-body {
-  background-color: #0277bd;
+.form-stripped div:nth-child(even){
```

Comment:
I think here is a typo, you mean *striped* right? not stripped like naked

Reply:
you are right.